elastalert/util.py

```diff
@@ -362,6 +362,7 @@ def build_es_conn_config(conf):
     parsed_conf['profile'] = None
     parsed_conf['headers'] = None
     parsed_conf['es_host'] = os.environ.get('ES_HOST', conf['es_host'])
+    parsed_conf['es_hosts'] = os.environ.get('ES_HOSTS', conf.get('es_hosts'))
```

Comment:
This you'll want to treat as a CVS if you want to support this in the first draft given that environment variables are strings